_episodes/11-hpc-intro.md

```diff
@@ -19,12 +19,6 @@ keypoints:
 
 ## Why Use These Computers?
 
-> ## What do you need?
->
-> Talk to your neighbor about your research. How does computing help you do your research? How could
-> more computing help you do more or better research?
-{: .challenge}
-
```

Comment:
I think it's important to motivate the connection between people's work (which they really care about) and what more computers can do for them.  If not here, via this exercise, it should be somewhere else. Is it already?  If not, we need to make sure this idea is not lost.

Reply:
ok, I get the point. Did that with ce3e91c